packages/ui/src/components/Dialog/LargeDialog.css.ts

```diff
@@ -0,0 +1,40 @@
+import { style } from '@vanilla-extract/css'
+import { minWidth, tokens } from '../../theme'
+
+export const closeButton = style({
+  position: 'absolute',
+  right: tokens.space.md,
+  top: tokens.space.md,
```

Comment:
Update with if updating padding, see comment below
```suggestion
  right: tokens.space.lg,
  top: tokens.space.lg,
```